crates/sui-gateway/src/unit_tests/rpc_server_tests.rs

```diff
@@ -109,7 +109,7 @@ async fn test_publish() -> Result<(), anyhow::Error> {
         .await?;
 
     let response = tx_response.to_publish_response()?;
-    assert_eq!(5, response.created_objects.len());
+    assert_eq!(6, response.created_objects.len());
```

Comment:
what causes this change?

Reply:
Me adding another example module in `fungible_tokens` which had an initializer creating an extra object. Now removed since we no longer really need an example (all coins should now be using one-time witness).